wpimath/src/test/native/cpp/interpolatable/TimeInterpolatableBufferTest.cpp

```diff
@@ -0,0 +1,24 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2020 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include <units/time.h>
+
+#include "frc/geometry/Rotation2d.h"
+#include "frc/interpolatable/TimeInterpolatableBuffer.h"
+#include "gtest/gtest.h"
+
+TEST(TimeInterpolatableBufferTest, TestInterpolation) {
```

Comment:
This needs to test that the correct entries are deleted when the history span is exceeded.